src/app/manage/media/media-overview/media-overview.component.html

```diff
@@ -30,20 +30,57 @@
     </app-overview-item>
 
   </app-card-overview>
+
+  <div class="mo__casual-overview" *ngSwitchCase="OVERVIEW_MODES.casual">
+    <div *ngIf="showGettingStarted$ | async" class="getting-started-info-pane">
+      <div>
+        <mat-icon inline svgIcon="info" color="primary" class="getting-started-info-pane__icon"></mat-icon>
+        <span>To add media files click on the
+          <button mat-fab color="warn" class="info-pane-inline-btn"><mat-icon svgIcon="add"></mat-icon></button>
```

Comment:
could this be without pointer events ? otherwise I'm pretty sure some people would click and wonder why I didnt do anything :D

Reply:
Also could you add like in `app-getting-started` an offline mode button (which will be used on github pages preview, since it doesn't have any server for it)

Once PR previews are added this will be useful